Modules/posixmodule.c

```diff
@@ -8279,7 +8282,7 @@ win32_getppid()
     pe.dwSize = sizeof(pe);
     have_record = Process32First(snapshot, &pe);
     while (have_record) {
-        if (mypid == (pid_t)pe.th32ProcessID) {
+        if (mypid == pe.th32ProcessID) {
             /* We could cache the ulong value in a static variable. */
             result = PyLong_FromPid((pid_t)pe.th32ParentProcessID);
```

Comment:
Likewise, let's just convert from DWORD, rather than using a C cast.

Reply:
I got rid of the cast, so I think this does not apply.

Reply:
Sorry, my comment is on a nearby line, not the one that you changed.

Reply:
:+1: applied